google/apitools
# Fix lint errors in apitools.

This makes a few updates to get the linter happy:

* drop oauth2l and oauth2l_test
* add some newlines
* fix formatting in two error messages
* switch `pep8` -> `pycodestyle`
* drop the custom pylintrc and `run_pylint`.

The "have a custom system for pylint" seemed like a good idea long ago, but
python linting tools have come far enough that it's more work than it's worth.

In particular, this should make travis go green again.

Fixes #200 

PTAL @znewman01 @vilasj
